uktrade/data-hub-frontend
# Refactored ToggleSection component variants

## Description of change

Refactors the toggle section to use style composition.

## Test instructions

Everything should look the same - check the toggle components on storybook, the new personalised dashboard and any of the listing pages.

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
